00_Project/sos/SERVICES/sos/sos_config.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * sos_config.h
+ *
+ * Created: 5/29/2023 8:24:51 PM
+ *  Author: sharb
+ */ 
+
+
+#ifndef SOS_CONFIG_H_
+#define SOS_CONFIG_H_
+
+
+typedef enum 
+{
+	TIMER_ZERO,
+	TIMER_ONE,
+	TIMER_TWO,
+}enu_sos_timer_select_t;
```

Comment:
Why you want the user to know anything about the timers?
This is a specific information that the application doesn't need to know about. As your system shall reserves a specific timer for the tick time.

Reply:
Done